libs/transfer/src/lib/components/make-transfer-summary/make-transfer-summary.component.ts

```diff
@@ -10,7 +17,10 @@ export class MakeTransferSummaryComponent {
   @Output() submitTransfer = new EventEmitter<void>();
   @Output() closeTransfer = new EventEmitter<void>();
 
+  constructor(@Optional() private readonly tracker?: Tracker) {}
+
   submit(): void {
+    this.tracker?.publish(new TransferSubmitEvent({}));
```

Comment:
Better put it in a smart-component implementation, where the `submitTransfer` output is handled.